spec/swagger.yaml

```diff
@@ -747,71 +646,84 @@ paths:
-          description: Patient id to fetch plan summary for
-          required: true
-        - name: include
-          description: List of related resources to include in the response
+          description: Patient id to fetch calendar event (either a patient, group, organization or attendees filter is required)
+        - name: filter[groups]
           in: query
-          required: false
           type: string
-          enum:
-            - actions
-            - bundles
-            - patient
-            - current_results
-      responses:
-        '200':
-          description: OK
-          schema:
-            $ref: '#/definitions/FetchPatientPlanSummariesResponse'
-        '401':
-          description: Unauthorized
-          schema:
-            $ref: '#/definitions/FetchErrorResponse'
-        '403':
-          description: Forbidden
-          schema:
-            $ref: '#/definitions/FetchErrorResponse'
-  /patient_plan_summary/{id}:
-    get:
-      tags:
-        - plan
-      summary: Get the plan summary for a patient
-      description: Get the plan summary for a patient.
-      operationId: fetchPatientPlanSummary
-      parameters:
-        - name: id
-          in: path
-          description: Plan summary identifier
-          required: true
+          description: Comma-separated list of Twine group ids (either a patient, group, organization or attendees filter is required)
+        - name: filter[organization]
+          in: query
+          type: string
+          description: Twine organization id (either a patient, group, organization or attendees filter is required)
+        - name: filter[attendees]
+          in: query
+          type: string
+          description: Comma-separated list of Twine coach or patient ids (either a patient, group, organization or attendees filter is required)
+        - name: filter[type]
+          in: query
+          type: string
+          description: Calendar event type. Valid values are `plan-check-in`, `reminder`, `telephone-call`, `office-visit` and `video-call`.
+        - name: filter[completed]
+          in: query
+          type: boolean
+          description: If not specified, return all calendar events. If set to 'true' return only events marked as completed, otherwise if set to 'false', return only events not marked as completed yet.
+        - name: filter[start_at]
+          in: query
+          type: string
+          description: The start (inclusive) and end (exclusive) date separated by a comman to filter events by the `start_at` date. Example for including all calendar events for November 2017 `filter[start_at]=11/01/2017,12/01/2017`.
+        - name: filter[end_at]
+          in: query
+          type: string
+          description: The start (inclusive) and end (exclusive) date separated by a comman to filter events by the `end_at` date. Example for including all calendar events for November 2017 `filter[end_at]=11/01/2017,12/01/2017`.
```

Comment:
"The start" should be "The end".

Reply:
Also, see above about dates and "comman".

Reply:
@sgilroy `start` in this context is the `beginning` of the date range, not the field being filtered on